test/functional/feature_nulldummy.py

```diff
@@ -82,31 +91,45 @@ def run_test(self):
         self.lastblocktime = int(time.time()) + self.lastblockheight
 
         self.log.info(f"Test 1: NULLDUMMY compliant base transactions should be accepted to mempool and mined before activation [{COINBASE_MATURITY + 3}]")
-        test1txs = [create_transaction(self.nodes[0], coinbase_txid[0], self.ms_address, amount=49)]
+        test1txs = [self.create_transaction(txid=coinbase_txid[0], addr=self.ms_address, amount=49,
+                                            privkey=self.nodes[0].PRIV_KEYS[0][1])]
         txid1 = self.nodes[0].sendrawtransaction(test1txs[0].serialize_with_witness().hex(), 0)
-        test1txs.append(create_transaction(self.nodes[0], txid1, self.ms_address, amount=48))
+        spk = test1txs[0].vout[0].scriptPubKey.hex()
+        test1txs.append(self.create_transaction(txid=txid1, prev_utxo_details={"scriptPubKey": spk,
+                                                "redeemScript": cms["redeemScript"]},
```

Comment:
nit: As the same `prev_utxo_details` are used more than once, I think it would be beneficial to the test's readability if you assigned them at the beginning of the test with something like 
`ms_address_unlock_details = {"scriptPubKey": spk,
                                                "redeemScript": cms["redeemScript"]}`